profile/legacy_profile.go

```diff
@@ -541,7 +541,7 @@ func parseHeap(b []byte) (p *Profile, err error) {
 		p.Sample = append(p.Sample, &Sample{
 			Value:    value,
 			Location: sloc,
-			NumLabel: map[string][]int64{"bytes": {blocksize}},
+			NumLabel: map[string]NumValues{"bytes": {Unit: "bytes", Values: []int64{blocksize}}},
```

Comment:
Should we now call this "size" instead of bytes?

Reply:
That would make sense to me. 
I did keep this as bytes for now to be consistent with other profiles being generated. 
Should I change this  to "size" anyways?

Reply:
I'd like to avoid "bytes"/"bytes" as it makes little sense. "size"/"bytes", or just continue to generate "bytes"/"" as before might be preferable -- we will have to deal with profiles generated by handles with "bytes"/"" for the foreseeable future anyway.

Reply:
I had briefly changed the key to "size", but then realized that graphs display information from the "bytes" tag. So, while I agree "size""bytes" would make more sense, I think making a change to this tag name might require thinking a bit more about how to use numeric tags in graphs.

I also think that the units field should be set within pprof. Otherwise, it would be necessary to infer units in multiple places in the code, (I know it would be necessary at least within tag filtering and within some of the code for different graphs), which doesn't seem as maintainable. Would it be clear enough if code were modified so that units were not written out if the profile were saved?

Reply:
Yes, bytes/bytes is confusing, and is being introduced here.

I'd rather either go to "size"/"bytes", or if we need more thinking, leave as "bytes"/"" for now. We need to continue supporting that combination for the foreseeable future anyway.

Reply:
I've modified this so it will be "bytes"/"" for now.